<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clockForm</class>
 <widget class="QWidget" name="clockForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>206</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    border-radius: 5px;
}
#frame{
	order:2px solid rgba(255,255,255,0.5);
}

#dayLabel,
#clockLabel,
#dateLabel {
	color: white;
	text-align: center;
}

#frame { 
    background-color: rgba(80,80,80, 0.2);
    border-radius: 10px;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
    border-radius: 5px;
}
#frame{
border:2px solid rgba(255,255,255,0.5);
}

#dayLabel,
#clockLabel,
#dateLabel {
	color: white;
	text-align: center;
}

#frame { 
    background-color: rgba(80,80,80, 0.2);
    border-radius: 10px;
}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="clockLabel">
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
          <pointsize>30</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>13:56</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dayLabel">
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string>Cumartesi</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dateLabel">
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>21.15.18</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
